tests/test_linters.py

```diff
@@ -16,27 +16,15 @@
 
 """Tests for analyze and lint code."""
 
-import sys
 import pathlib
+import sys
 from unittest.mock import patch
 
 import pytest
-
-from charmcraft.linters import (
-    CHECKERS,
-    CheckType,
-    FATAL,
-    Framework,
-    IGNORED,
-    JujuActions,
-    JujuConfig,
-    JujuMetadata,
-    Language,
-    UNKNOWN,
-    analyze,
-    check_dispatch_with_python_entrypoint,
-)
-
+from charmcraft.linters import (CHECKERS, FATAL, IGNORED, UNKNOWN, CheckType,
```

Comment:
This format change is because a new black? Can we stick to the old format, please?

Furthermore, why this part of the diff anyway? I don't see any changes in the import list (and comparing them is something this new format makes very difficult).

Reply:
New black, yes. Reverted for now